File: Day-11/oc_of_4.py
t = int(input("\n Enter No of TestCases: "))
for _ in range(t):
    print("\n\n TestCase {}:".format(_+1))
    n = int(input("\n Enter an Integer: "))
    temp = n
    count = 0
    while n > 0:
        digit = n % 10
        if digit == 4:
            count += 1
        n //= 10
    print(" Occurrences of 4 in {} is: {}".format(temp, count))

File: Day-11/oc_of_4.c
#include <stdio.h>

int main(void) {
	// your code goes here
	int t;
    printf("Enter No of TestCase: ");
	scanf("%d", &t);
	while(t--) 
	{
	    int n, count = 0, digit;
        printf("\n\nEnter the Integer: ");
	    scanf("%d", &n);
        int temp = n;
	    while(n > 0) {
	        digit = n % 10;
	        if(digit == 4)
	            count++;
	        n /= 10;
	    }
	    printf("\nThe Occurrences of 4 in %d is: %d", temp, count);
	}
	return 0;
}

